composer.json

```diff
@@ -27,6 +27,7 @@
         "doctrine/doctrine-bundle": "^2.0",
         "doctrine/doctrine-migrations-bundle": "^2.2|^3.0",
         "doctrine/mongodb-odm-bundle": "^3.1|^4.2",
+        "doctrine/mongodb-odm": "^2.0",
```

Comment:
this prevent some weird deprecations triggered by a `dev-*` version of `doctrine/mongo-odm`